tests/functional/aws-node-sdk/test/multipleBackend/delete.js

```diff
@@ -64,7 +71,18 @@ describe('Multiple backend delete', () => {
                     `Expected success, got error ${JSON.stringify(err)}`);
                 s3.getObject({ Bucket: bucket, Key: fileObject }, err => {
                     assert.strictEqual(err.code, 'NoSuchKey', 'Expected ' +
-                        'error but got sucess');
+                        'error but got success');
+                    done();
+                });
+            });
+        });
+        it('should delete object from AWS', done => {
+            s3.deleteObject({ Bucket: bucket, Key: awsObject }, err => {
```

Comment:
You don't have to define the location constraint when getting or deleting an object? What happens if you have objects put with the same names in buckets of the same name in different backends?